tensorly/regression/cp_plsr.py

```diff
@@ -97,6 +99,7 @@ def fit(self, X, Y):
         self.Y_factors = [
             T.zeros((l, self.n_components), **T.context(X)) for l in T.shape(Y)
         ]
+        self.coef_ = T.zeros((self.n_components, self.n_components), **T.context(X))
```

Comment:
What is `coef_`? Can you provide a comment?

Reply:
These are the coefficients of the linear model, the same as $\beta$ in ordinary least square $Y = X \beta + \epsilon$. This was notated in Bro's paper as $\mathbf{b}$. This parameter has always existed in Bro's paper, but was previously dismissed by me as trivial since there was no explanation for it.

Reply:
This is quite useful -- could we add it to the docstrings (e.g. in Notes) or the doc?